laravel-app/app/Http/Controllers/PostEditController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\UserPostRequest;
+use App\Models\User;
+use App\Models\UserPost;
+use App\Services\PostService;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Facades\Auth;
+
+class PostEditController extends Controller
+{
+    protected PostService $postService;
+
+    public function __construct(PostService $postService)
+    {
+        $this->postService = $postService;
+    }
+    public function index()
+    {
+        return view ();
+    }
+    public function show(UserPost $post)
+    {
+        // dd($post);
```

Comment:
remove this please

Reply:
I forgot to remove the whole PostEditController since its not needed po. Instead I appended update() in PostController